tests/subsys/zigbee/zboss_api/callback_api/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+cmake_minimum_required(VERSION 3.8)
+
+add_definitions(-Wno-packed-bitfield-compat)
+add_definitions(-DTIMER_DEFAULT_CONFIG_IRQ_PRIORITY=6)
```

Comment:
This looks like something that should have been in a configuration file.
Kconfig ?
Secondly, I can't find the place where it is used.

Reply:
It's not needed any more. Removed